installing/Ubuntu.md

```diff
@@ -0,0 +1,51 @@
+# Ubuntu
+
+We chose Ubuntu trusty (14.04) for our example, but newer and older
+versions would work just the same.
+
+First some required dependencies:
+
+```bash
+sudo apt-get install libdbd-sqlite3-perl libdbi-perl libwww-curl-perl
+```
+
+We use the *incredibly dangerous and generally discouraged* way of
+piping a random shell script from the Internet into bash. We do this
+because we trust the Nix people.
+
+```bash
+bash <(curl https://nixos.org/nix/install)
```

Comment:
There are but there's a bit of permission fiddling, bash-curl is by far the easiest way.